lib/Conversation/styles.scss

```diff
@@ -1,9 +1,16 @@
 /* ==========================================================================
    Config
    ========================================================================== */
+$conversation-active-background-color: #ffffff !default;
+$conversation-active-shadow-color: rgba(158, 158, 158, 0.17) !default;
```

Comment:
Is `rgba(158, 158, 158, ...)` based off an existing setting? If so, maybe `rgba($setting-name, ...)` 👍